writer/writer/core/writer_service.py

```diff
@@ -33,10 +33,6 @@ def write(
             with self.database.get_context():
                 # Convert request events to db events
                 self.db_events = self.event_translator.translate(write_request.events)
-                if not self.db_events:
```

Comment:
Why did you remove this? Can it not happen?

Reply:
No, I added it in the previous PR because some list updates (e.g. a "remove" on an empty field) would yield no events. With the new `DbListUpdateEvent`, it *had* to to that, so it now works lik a `DeleteFieldsEvent` on a non-existing field (or, t be more specific, it creates one).